src/TC/src/FfmpegDecoder.cpp

```diff
@@ -196,32 +202,6 @@ struct FfmpegDecodeFrame_Impl {
     return true;
   }
 
-  bool SaveGRAY12LE(AVFrame* pframe) {
-    size_t size = frame->width * frame->height * 2;
-
-    if (!dec_frame) {
-      dec_frame = Buffer::MakeOwnMem(size);
-    } else if (size != dec_frame->GetRawMemSize()) {
-      delete dec_frame;
-      dec_frame = Buffer::MakeOwnMem(size);
-    }
-
-    auto plane = 0U;
-    auto* dst = dec_frame->GetDataAs<uint8_t>();
-
-    auto* src = frame->data[plane];
-    auto width = frame->width;
-    auto height = frame->height;
-
-    for (int i = 0; i < height; i++) {
-      memcpy(dst, src, 2*width);
-      dst += 2*width;
-      src += frame->linesize[plane];
-    }
-
-    return true;
-  }
-
```

Comment:
Any reason for removing this ?

Reply:
That probably happened because I created PR, then #502 was merged, and only then I submitted #515
So the answer is no, this is unintentional.